lambda/checkExpiry/index.js

```diff
@@ -9,7 +9,8 @@ exports.handler = async (event, context) => {
     };
 
     // Look for today's expiries
-    let yesterdaysDate = formatDate(new Date())
+    let yesterdaysDate = new Date();
+    yesterdaysDate.setDate(yesterdaysDate.getDate()-1);
```

Comment:
Nice, that's a neat way to do it. I think it needs to be formatted still though.